test/integration/logs/benchmark/load_test.go

```diff
@@ -152,12 +152,12 @@ func TestData(t *testing.T) {
 					},
 				},
 			},
-			printf:     `[RemoteRuntimeService] Version Response: [apiVersion &VersionResponse{Version:0.1.0,RuntimeName:containerd,RuntimeVersion:v1.6.18,RuntimeApiVersion:v1,}]`,
-			structured: `"[RemoteRuntimeService] Version Response" apiVersion="&VersionResponse{Version:0.1.0,RuntimeName:containerd,RuntimeVersion:v1.6.18,RuntimeApiVersion:v1,}"`,
+			printf:     `[RemoteRuntimeService] Version Response: [apiVersion version:"0.1.0"  runtime_name:"containerd"  runtime_version:"v1.6.18"  runtime_api_version:"v1"]`,
```

Comment:
hmm... proto might output non-deterministic content here ... I saw both `" "` and `"  "` spacing when running the tests :-/

Reply:
`#String()` going from deterministic to non-deterministic is going to be a *super* fun edge to deal with generally as part of moving away from gogo :-/

Reply:
Thanks, I applied the patch as suggested.